templates/zerver/app/compose.html

```diff
@@ -6,42 +6,43 @@
                 <span id="compose-reply-error-msg"></span>
             </div>
             <div id="compose_buttons">
+                <span class="new_message_button reply_button_container">
+                    <button type="button" class="button small rounded compose_reply_button"
+                      id="left_bar_compose_reply_button_big"
+                      title="{{ _('Reply') }} (r)">
+                        <span class="compose_reply_button_label">{{ _('Reply to') }}&nbsp;<span class="compose_reply_button_recipient_label"></span>
```

Comment:
Translation should always be applied to a full string with a placeholder, not to a fragment, to allow translators to change the word order as necessary for their language.

The way to do this in our setup is with a `{{#tr}}` block:

```handlebars
<span class="compose_reply_button_label">
    {{tr}}
        Reply to <z-recipient></z-recipient>
        {{#*inline "z-recipient"}}<span class="compose_reply_button_recipient_label"></span>{{/inline}}
    {{/tr}}
</span>
```

Additionally, the “Reply to #stream > topic” text is inconsistent with the “Message #stream > topic” that it turns into when you click on it.

Reply:
We're likely moving this file to a frontend template: https://github.com/zulip/zulip/pull/18154; depending how that serializes with this things may be simplified.

I think I agree we should go with "Message ..." as the text, to match the placeholder.

Reply:
The second example ("{{ _("Reply to {recipient}").format...") appears to work. I'll push that to here soon.

I think changing it to "Message " is a fine idea. In the longer term, would that imply that the 'r' shortcut should become 'm'?

Reply:
Unclear -- arguably `r` replies to the selected message, and this button sends a message to the place it says it will, which happens to be replying to the selected message?